configPublishPath

```diff
@@ -0,0 +1,10 @@
+{
+  "Microsoft.PowerShell:ExecutionPolicy": "RemoteSigned",
+  "ExperimentalFeatures": [
+    "Microsoft.PowerShell.Utility.PSDebugRunspaceWithBreakpoints",
+    "PSCommandNotFoundSuggestion",
+    "PSImplicitRemotingBatching",
+    "PSTempDrive",
+    "PSUseAbbreviationExpansion"
```

Comment:
It seems last two is already not experimental.

Reply:
Saw that and should be fixed already with latest commit